<launch>

    <!-- <node pkg="joy" name="joy" type="joy_node" args="/dev/input/js2">
        <param name="dev" type="string" value="/dev/input/js2" />
    </node> -->

    <node pkg="rosserial_arduino" name="serial_node" type="serial_node.py" >
        <param name="port" value="/dev/ttyUSB0" />
        <param name="baud" value="115200" />
    </node>

    <node pkg="robot_arm" name="routine" type="routine.py" />
    <node pkg="robot_arm" name="esp32_ctrl" type="esp32_controller.py" />

</launch>